controller/aks-cluster-config-handler.go

```diff
@@ -50,8 +50,8 @@ const (
 	// node software, return code 3
 	ClusterStatusInProgress = "InProgress"
 
-	// ClusterStatusUpgrading The Upgrading state indicates the cluster is updating
-	ClusterStatusUpgrading = "Upgrading"
+	// ClusterStatusUpdating The Updating state indicates the cluster is updating
+	ClusterStatusUpdating = "Updating"
```

Comment:
Looks like this is purely grammatical, no functional change.

Reply:
This does change the behavior in that the operator actually waits to apply changes now.  Before it would log constant error messages because you tried to patch a cluster in an updating state.

Reply:
Talked offline. There aren't good docs around this, but was proved out with some testing.